### PR TITLE
Update/procurations

### PR DESCRIPTION
**Pull request by marineb**

### CODE REVIEW

File: src/AppBundle/Entity/ProcurationRequest.php [hunk 1]
```diff
@@ -272,6 +272,13 @@ class ProcurationRequest
     /**
      * @var bool
      *
+     * @Assert\IsTrue(message="procuration.authorization.required", groups={"front"})
+     */
+    public $authorization;
```
Comment (HeahDude) [inline src/AppBundle/Entity/ProcurationRequest.php]: We do not need such public property on the entity, we can use the `mapped` option set to `false` on the form fields instead.

**Comment by marineb** [comment]
@HeahDude Made an update. Let me know if that's what you meant / work for you!

**Comment by tgalopin** [comment]
I think that's not what he meant. Can you either explain more or even add a commit in the PR @HeahDude ?

**Comment by HeahDude** [comment]
Done, I've just pushed https://github.com/EnMarche/en-marche.fr/pull/901/commits/2972501105ce46527fcb481fd7ecf8d04c0fc974.

**Comment by HeahDude** [comment]
Ok, it's green now, I've fixed a bug in validation groups while I was at it ;).

**Pull request merged**